tests/testing_utils/files_and_dirs.py

```diff
@@ -31,6 +31,7 @@ def create_temp_file(suffix: str, dir_name: str, contents: List[str]) -> str:
 
 def create_named_file(file_name: str, dir_name: str, contents: List[str]):
     file_path = os.path.join(dir_name, file_name)
+    os.makedirs(os.path.dirname(file_path), exist_ok=True)
```

Comment:
doesn't need to be here. We're co-owners, so no CLI approval needed. I'd remove it from this PR.